src/haz3lweb/app/explainthis/ExplainThis.re

```diff
@@ -631,6 +631,7 @@ let get_doc =
         let pat_id = List.nth(pat.ids, 0);
         let body_id = List.nth(body.ids, 0);
         switch (pat.term) {
+        // TODO (Anthony): put in a real message
```

Comment:
@WondAli I haven't looked at any of this file so it would be good if you double checked to see if anything remains.

Reply:
remove